examples/programmatic/llpr_forces/force_llpr.py

```diff
@@ -183,11 +183,12 @@
 force_uncertainties = []
 
 for batch in test_dataloader:
+    dtype = getattr(torch, model.capabilities().dtype)
```

Comment:
That's clever =D

Reply:
Everything's easy when you don't have to torchscript ^___^